examples/ucp_hello_world.c

```diff
@@ -400,7 +400,6 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
     if (err_handling_opt.failure_mode == FAILURE_MODE_SEND) {
         fprintf(stderr, "Emulating unexpected failure on server side, client "
                         "should detect error by keepalive mechanism\n");
-        free(msg);
```

Comment:
can we add exit() after raise() instead, since exit() has __noreturn__ attribute

Reply:
with exit(1) to indicate an error happened?

Reply:
yes